CONTRIBUTING.rst

```diff
@@ -21,15 +21,26 @@ Report a `new issue <https://github.com/mozilla/pontoon/issues/new>`_:
 Docker
 ======
 
-Everything runs in a Docker container. Thus Pontoon requires fewer things to get
-started and you're guaranteed to have the same setup as everyone else and it
-solves some other problems, too.
+While the front-end (JavaScript) build and tests use the host environment for development,
+the back-end systems (Python/Django, databases, etc.) run in Docker containers.
+For production use, also the front-end is built in a container.
+Thus Pontoon requires fewer things to get started
+and you're guaranteed to have the same server setup as everyone else.
 
 If you're not familiar with `Docker <https://docs.docker.com/>`_ and
 `docker-compose <https://docs.docker.com/compose/overview/>`_, it's worth
 reading up on.
 
 
+JavaScript setup
+================
+
+For working on the front-end, you need at least Node.js 14 and npm 7
+(`installation instructions <https://docs.npmjs.com/downloading-and-installing-node-js-and-npm>`_).
+Parts of the front-end use `npm workspaces <https://docs.npmjs.com/cli/v7/using-npm/workspaces>`_,
+which are not supported by earlier npm versions.
```

Comment:
I don't know if we need to call this out here, but we should update both Prerequisites sections: 
https://mozilla-pontoon.readthedocs.io/en/latest/dev/setup.html#prerequisites
https://mozilla-pontoon.readthedocs.io/en/latest/dev/setup-virtualenv.html#prerequisites

Reply:
Ah, yeah. Will fix that here.